cpp/src/datetime/datetime_ops.cu

```diff
@@ -442,6 +534,50 @@ gdf_error gdf_extract_datetime_day(gdf_column *input, gdf_column *output) {
 	return GDF_SUCCESS;
 }
 
+gdf_error gdf_extract_datetime_weekday(gdf_column *input, gdf_column *output) {
+
+	GDF_REQUIRE(input->size == output->size, GDF_COLUMN_SIZE_MISMATCH);
+	GDF_REQUIRE(output->dtype == GDF_INT16, GDF_UNSUPPORTED_DTYPE);
+
+	cudaStream_t stream;
+	cudaStreamCreate(&stream);
+
+
+    if (input->valid){
+      gdf_size_type num_bitmask_elements = ( ( input->size +( GDF_VALID_BITSIZE - 1)) / GDF_VALID_BITSIZE );
```

Comment:
I am not actually sure what this line of code does, I just used the other fields as a template essentially. So let me know if this is wrong.

Reply:
Yikes, code like this shouldn't still exist :crying_cat_face: 

Use https://github.com/rapidsai/cudf/blob/12f15c81c36659cdec2935cfec38b02968a4deda/cpp/include/cudf/functions.h#L76 instead.

Reply:
fixed.